pkg/codegen/python/gen_program.go

```diff
@@ -553,12 +553,14 @@ func (g *generator) genConfigVariable(w io.Writer, v *pcl.ConfigVariable) {
 	g.genTemps(w, temps)
 
 	name := PyName(v.Name())
+<<<<<<< HEAD
```

Comment:
I don't think this merge was finished.

Reply:
Oops - I was rebasing this branch by mistake and ended up pushing it half-way.  Let me tidy it up.